common/sketch/src/main/java/org/apache/spark/util/sketch/CountMinSketchImpl.java

```diff
@@ -17,17 +17,37 @@
 
 package org.apache.spark.util.sketch;
 
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
 import java.io.DataInputStream;
 import java.io.DataOutputStream;
+import java.io.Externalizable;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
 import java.io.OutputStream;
 import java.io.UnsupportedEncodingException;
 import java.util.Arrays;
 import java.util.Random;
 
-class CountMinSketchImpl extends CountMinSketch {
-  public static final long PRIME_MODULUS = (1L << 31) - 1;
+/*
+ * Binary format of a serialized CountMinSketchImpl, version 1 (all values written in big-endian
```

Comment:
This comment has been moved to `CountMinSketch.Version` as @rxin suggested in https://github.com/apache/spark/pull/10920#discussion_r50802512